alpaca/rest_test.go

```diff
@@ -443,6 +443,192 @@ func TestClient_GetAnnouncement(t *testing.T) {
 	require.NotNil(t, announcement)
 }
 
+func TestClient_GetWatchlists(t *testing.T) {
+	c := testClient()
+	// successful
+	c.do = func(c *client, req *http.Request) (*http.Response, error) {
+		assert.Equal(t, "api.alpaca.markets", req.URL.Host)
+		assert.Equal(t, "/v2/watchlists", req.URL.Path)
+		assert.Equal(t, "GET", req.Method)
+
+		watchlists := []Watchlist{
+			{
+				AccountID: "123",
+				ID:        "some_id",
+				Name:      "testname",
+				Assets: []Asset{
+					{
+						ID:       "some_id",
+						Name:     "AAPL",
+						Exchange: "NASDAQ",
+					},
+				},
+			},
+		}
+
+		return &http.Response{
+			Body: genBody(watchlists),
+		}, nil
+	}
+
+	watchlists, err := c.GetWatchlists()
+	require.NoError(t, err)
+	require.Len(t, watchlists, 1)
+}
```

Comment:
I would also assert that at least some of the fields of the watchlist actually matches.

Reply:
I have added the fix in the latest commit 😄